src/Pages/ProductInfoPage/ProductSub.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import './ProductSub.scss';
+import ProductSubHeader from './ProductSubHeader';
+import ProductSubItem from './ProductSubItem';
+
+export default class ProductSub extends Component {
+  render() {
+    return (
+      <div className="productSub">
+        <ProductSubHeader />
+        <div className="subItemBox">
+          {this.props.subItemList.map(el => {
```

Comment:
```suggestion
          {this.props.subItemList.map(subItem => {
```
el, data 등 불명확한 이름은 피해주세요

Reply:
넵 모두 수정했습니다.!